modules/gcp_k8s_service/gcp-k8s-service.md

```diff
@@ -88,3 +88,30 @@ underlying resource kind is being switched.
 _NOTE_ because of the nature of these disks, they will not be cleaned up automatically unless during a service
 destruction. If you wish to release the persistent disks for whatever reason you will need to manually do so by deleting
 the kubernetes persistent volume claims.
+
+### Tolerations
+
+Opta now gives you the option of adding [taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+to the nodes created in this nodepool, and thusly the ability to add _tolerations_ for said taints. The official
+documentation gives an excellent detailed summary, but in short one can use taints to stop workloads from running in
+said nodes unless they have a matching toleration. Simply provide a list of desired tolerations as inputs like so:
+```yaml
+  - name: app
+    type: k8s-service
+    image: AUTO
+    healthcheck_path: "/get"
+    port:
+      http: 80
+    tolerations:
+      - key: blah1
+        value: baloney
+        effect: "NoExecute"
+      - key: blah2
+        value: baloney
+        # Tolerates for default effect of NoSchedule
+      - key: blah3
```

Comment:
Opta assumes it can install some helmm charts (e.g. ingress); if a user adds taints to the "default" nodepool then opta may not be able to function correctly (e.g. some user adds no-execute to the default pool); we should inform the user about Opta needing a "taintfree" nodepool to function properly...

Reply:
@bigbitbus excellent question! That's why I'm not allowing taints on the "default" nodepools lol (those are created in the eks/gke module and it does not have this new feature)